docs/modules/ROOT/pages/how-tos/cloudscale/decommission.adoc

```diff
@@ -0,0 +1,124 @@
+= Uninstallation on cloudscale.ch
+
+[abstract]
+--
+Steps to remove an OpenShift 4 cluster from https://cloudscale.ch[cloudscale.ch].
+--
+
+[NOTE]
+--
+The commands are idempotent and can be retried if any of the steps fail.
+--
+
+
+== Prerequisites
+
+* `docker`
+* `mc` https://docs.min.io/docs/minio-client-quickstart-guide.html[Minio client] (aliased to `mc` if necessary)
+* `jq`
+* `yq` https://mikefarah.gitbook.io/yq[yq YAML processor]
+
+
+== Cluster Decommission
+
+. Export the following vars
++
+[source,console]
+----
+export CLOUDSCALE_TOKEN=<cloudscale-api-token> # From https://control.cloudscale.ch/user/api-tokens
+export CLUSTER_ID=<lieutenant-cluster-id>
+export REGION=<region> # rma or lpg (without the zone number)
+export GITLAB_TOKEN=<gitlab-api-token> # From https://git.vshn.net/profile/personal_access_tokens
+export GITLAB_CATALOG_PROJECT_ID=<project-id> # GitLab numerical project ID of the catalog repo
+----
+
+. Setup Terraform
++
+Initiate access to s3 bucket and initiate terraform
++
+[source,console]
+----
+# Use already exiting bucket user
+response=$(curl -sH "Authorization: Bearer ${CLOUDSCALE_TOKEN}" \
+  https://api.cloudscale.ch/v1/objects-users | \
+  jq -e ".[] | select(.display_name == \"${CLUSTER_ID}\")")
+
+# configure minio client to use the bucket
+mc config host add \
+  "${CLUSTER_ID}" "https://objects.${REGION}.cloudscale.ch" \
+  $(echo $response | jq -r '.keys[0].access_key') \
+  $(echo $response | jq -r '.keys[0].secret_key')
+
+# list/confirm contentes of the bucket
+mc tree ${CLUSTER_ID}
+
+# Get ignition_bootstrap token from bucket
```

Comment:
This step can be skipped.

Reply:
I agree, it was a step to be sure you are in the correct bucket.
But you will receive that message when you config mc with the host